src/sass/components/_repo-images.scss

```diff
@@ -0,0 +1,42 @@
+// Repo Images
+
+// Variables specific to repo images
+$repo-img-size: calc(3.5rem + 3vw);
+$repo-img-size-static: 4.75rem;
+$repo-img-pos-adjust: -1.125rem;
+
+.ms-repo-item {
+  // increase default height/weight of icon in repo visualpicker
+  .slds-media__figure {
+    > div {
+      height: $repo-img-size;
+      width: $repo-img-size;
+
+      .slds-avatar_large {
+        height: 100%;
+        width: 100%;
+        font-size: $font-size-10;
+        font-weight: $font-weight-bold;
+      }
+
+      abbr {
```

Comment:
Similar to the bolding, I'm curious why we remove text-shadow here? 

Either way, I'd expect this nested in `.ms-repo-item .slds-avatar_large`, but not sure it needs to be in `.slds-media__figure > div`

Reply:
yeah, I should have caught this. I will remove.